.github/workflows/build.yml

```diff
@@ -0,0 +1,63 @@
+name: build
+
+on:
+  push:
+    branches: [ master, '[0-9]+.[0-9]+', '[0-9]+.[0-9]+.[0-9]+' ]
+  pull_request:
+    branches: [ master, '[0-9]+.[0-9]', '[0-9]+.[0-9]+.[0-9]+' ]
+
+jobs:
+  build:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    name: "PHP ${{ matrix.php-versions }} Test (deps: ${{ matrix.dependencies }})"
+
+    continue-on-error: ${{ matrix.experimental }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
+        dependencies: ['highest']
+        composer-arguments: [''] # to run --ignore-platform-reqs in experimental builds
+        experimental: [false]
+        include:
+          # Experimental build for PHP 8
+          #- php-versions: '8.0'
+          #  composer-arguments: '--ignore-platform-reqs'
+          #  static-analysis: 'no'
+          #  experimental: true
+          #  dependencies: 'highest'
+
+          # Build for minimum dependencies. Note: Installation with PHP 5.5 fails due to PHPUnit PHP requirements
+          - php-versions: '7.2'
```

Comment:
Would love to build this with PHP 5.5 to match lowest supported, but PHPUnit > 4.x won't install with such a low PHP version. So either don't test with that PHP version or write tests such that they are still compatible with PHPUnit 4.x

Reply:
That's because it allows/installs PHPUnit 4.8 https://github.com/TYPO3/html-sanitizer/runs/3421183462?check_suite_focus=true#step:6:20
Again, would require to make the tests in this project backwards-compatible with that old version of PHPUnit. Doable, but a lot of work for little benefit (PHP 5.5 wasn't tested with travis either)